OCR/frontend/src/componets/UploadFile.tsx

```diff
@@ -0,0 +1,22 @@
+import { FileInput, Icon } from "@trussworks/react-uswds"
+import { useId } from "react"
+
+export const Uploadfile = () => {
+    const id = useId()
+    return (
+        <div style={{ display: "flex", flexDirection: 'column', justifyContent: 'start', backgroundColor: '#d9e8f6', width: '100%', height: '100%', alignItems: 'center', paddingTop: '10px' }}>
+            <div style={{ width: '70%', textAlign: 'left' }}>
+                <h1 style={{  margin: 0 }}>Upload new form to segment</h1>
+                <h2 style={{ margin: 0 }}>Upload new image or PDF and save as a template</h2>
+            </div>
+            <div data-testid='dashed-container' style={{ display: 'flex', flexDirection: 'column',  width: '70%', height: '40%', border: '1px dashed  #005ea2', marginTop: '20px', justifyContent: 'space-around', alignItems: 'center', backgroundColor: '#ffffff' }}>
+                <Icon.UploadFile data-testid='upload-icon' style={{ marginTop: '16px' }} size={5} color="#005ea2" />
+                <div style={{ width: '60%', marginBottom: '8px' }}>
```

Comment:
I recommend adding this class
`className="display-flex flex-column flex-align-center"`

this will center the upload 
before
![image](https://github.com/user-attachments/assets/f38fc44e-b5c4-4d01-aefb-c7393d4a2e81)

after:
![image](https://github.com/user-attachments/assets/3b31f58b-4e85-422a-b920-777e6933fe28)